tilavarauspalvelu/integrations/email/template_context/application.py

```diff
@@ -1,28 +1,41 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, overload
 
 from django.utils.translation import pgettext
 
-from tilavarauspalvelu.translation import get_translated
+from tilavarauspalvelu.enums import WeekdayChoice
+from tilavarauspalvelu.models import RecurringReservation
+from tilavarauspalvelu.translation import get_attr_by_language, get_translated
 
 from .common import (
     create_anchor_tag,
     get_contex_for_base_template,
+    get_contex_for_closing,
     get_contex_for_closing_polite,
+    get_contex_for_closing_staff,
+    get_contex_for_seasonal_reservation_check_details_url,
     get_my_applications_ext_link,
+    get_staff_reservations_ext_link,
+    params_for_application_section_info,
+    params_for_reservation_series_info,
 )
 
 if TYPE_CHECKING:
+    from tilavarauspalvelu.models import ApplicationSection
     from tilavarauspalvelu.typing import EmailContext, Lang
 
 __all__ = [
     "get_context_for_application_handled",
     "get_context_for_application_in_allocation",
     "get_context_for_application_received",
+    "get_context_for_application_section_cancelled",
 ]
 
 
+# type: EmailType.APPLICATION_HANDLED ##################################################################################
```

Comment:
`# type:` is a special syntax for comments used by python type checkers to add information about typing (see [PEP484](https://peps.python.org/pep-0484/), e.g. `# type: ignore` for indicating that type checkers should ignore type errors for some code segment). Therefore I would use some other keyword here, e.g. `# email type: APPLICATION_HANDLED`

Reply:
My intention here pretty much to abuse the `# type: ` comment to have the IDE recognise the type and create a "link" directly to the Enum. Changing it to `# email type:` breaks this "link", but I also wouldn't want to break the standards 🤔

Reply:
You could use `Annotated[EmailContext, EmailType.APPLICATION_HANDLED]`, though I suppose it's a bit long. Well, it's not like there is anything wrong abusing the type comment like this so it's your call.

Reply:
Also, I think the fact that type comments create links like this might be a PyCharm specific thing, though I'm not sure.

Reply:
It's what we both use, and since this is not a big issue and makes the dividers prettier, lets keep it as-is for now